csharp/Platform.IO/FileHelpers.cs

```diff
@@ -20,6 +52,22 @@ public static T[] ReadAll<T>(string path)
             return reader.ReadAll<T>();
         }
 
+        /// <summary>
+        /// <para>Reads and returns <typeparamref name="T"/> struct from <paramref name="path"/>.</para>
+        /// <para>Считывае, и возвращает структуру типа <typeparamref name="T"/> из <paramref name="path"/>.</para>
+        /// </summary>
+        /// <typeparam name="T">
+        /// <para>The structure type.</para>
+        /// <para>Тип являющийся структурой.</para>
+        /// </typeparam>
+        /// <param name="path">
+        /// <para>The path to the file, from which to read the array of <typeparamref name="T"/> structures.</para>
```

Comment:
`the array of <typeparamref name="T"/> structures` ⇾ `the value of <typeparamref name="T"/> structure`